x/staking/keeper/delegation.go

```diff
@@ -259,16 +259,6 @@ func (k Keeper) IterateDelegatorRedelegations(ctx sdk.Context, delegator sdk.Acc
 	}
 }
 
-// HasMaxUnbondingDelegationEntries - check if unbonding delegation has maximum number of entries.
-func (k Keeper) HasMaxUnbondingDelegationEntries(ctx sdk.Context, delegatorAddr sdk.AccAddress, validatorAddr sdk.ValAddress) bool {
-	ubd, found := k.GetUnbondingDelegation(ctx, delegatorAddr, validatorAddr)
-	if !found {
-		return false
-	}
-
-	return len(ubd.Entries) >= int(k.MaxEntries(ctx))
-}
-
```

Comment:
Why is this removed, this should be there.

Reply:
Ok